circle.yml

```diff
@@ -1,3 +1,18 @@
+dependencies:
+  cache_directories:
+    - "~/cache"
+  pre:
+    # Only create directory if not already present (--parents option).
+    - mkdir --parents ~/cache
+  post:
+    # Only downloads if not already present (--no-clobber option)
+    - cd ~/cache && wget --no-clobber http://selenium-release.storage.googleapis.com/3.3/selenium-server-standalone-3.3.0.jar
+    - java -jar ~/cache/selenium-server-standalone-3.3.0.jar:
+        background: true
+test:
+  pre:
+    - npm start:
+        background: true
```

Comment:
I would recommend you wait for the server to start before starting the tests.

Suggested implementation:

```
   # wait for server
    - wget --retry-connrefused --waitretry=1 --output-document=/dev/null http://localhost:$PORT
```